runner/MODULE.bazel

```diff
@@ -49,7 +49,7 @@ single_version_override(
 # Go modules
 go_deps = use_extension("@gazelle//:extensions.bzl", "go_deps")
 go_deps.from_file(go_mod = "//:go.mod")
-use_repo(go_deps, "com_github_bazelbuild_buildtools", "com_github_fatih_color", "com_github_go_git_go_git_v5", "com_github_pmezard_go_difflib", "in_gopkg_yaml_v3", "io_opentelemetry_go_otel", "io_opentelemetry_go_otel_trace", "org_golang_x_term")
+use_repo(go_deps, "com_github_aspect_build_aspect_gazelle_common", "com_github_aspect_build_aspect_gazelle_language_js", "com_github_aspect_build_aspect_gazelle_language_kotlin", "com_github_aspect_build_aspect_gazelle_language_orion", "com_github_bazelbuild_bazel_gazelle", "com_github_bazelbuild_buildtools", "com_github_bufbuild_rules_buf", "com_github_fatih_color", "com_github_go_git_go_git_v5", "com_github_pmezard_go_difflib", "in_gopkg_yaml_v3", "io_opentelemetry_go_otel", "io_opentelemetry_go_otel_trace", "org_golang_x_term")
```

Comment:
You need to manually revert this, except the `rules_buf` part

Reply:
Something must have changed in rules_go to fix this, previously it wouldn't compile 🤷 👍